supabase.types.ts

```diff
@@ -189,6 +189,36 @@ export interface Database {
           }
         ]
       }
+      property_tags: {
+        Row: {
+          property_id: string
+          tag_id: string
+        }
+        Insert: {
+          property_id: string
+          tag_id: string
+        }
+        Update: {
+          property_id?: string
+          tag_id?: string
+        }
+        Relationships: [
+          {
+            foreignKeyName: "property_tags_property_id_fkey"
+            columns: ["property_id"]
+            isOneToOne: false
+            referencedRelation: "properties"
```

Comment:
i think the idea was supposed to be that tags are added to reviews, rather than directly to the properties. Then we can define a function to calculate how many of each tag has been set on reviews for a property.